zkbreacharbiter_test.go

```diff
@@ -263,8 +306,8 @@ func testZkBreachSpends(t *testing.T, test breachTest) {
 		t.Fatalf("breach arbiter didn't send ack back")
 	}
 
-	notifier := brar.cfg.Notifier.(*mockSpendNotifier)
-	notifier.confChannel <- &chainntnfs.TxConfirmation{}
+	// notifier := brar.cfg.Notifier.(*mockSpendNotifier)
```

Comment:
Is there a reason to leave this as a comment? Are we gonna uncomment it at some point?

Reply:
removed